src/components/userPage/UserDashboard.tsx

```diff
@@ -0,0 +1,51 @@
+import Box from '@mui/material/Box';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Divider from '@mui/material/Divider';
+import InboxIcon from '@mui/icons-material/Inbox';
+import DraftsIcon from '@mui/icons-material/Drafts';
+
+export default function BasicList() {
```

Comment:
Should it look like that?
![Zrzut ekranu 2024-04-13 o 16 22 40](https://github.com/rezio3/wave-trace/assets/17573378/91b1a206-df7d-4c8d-bdfb-4edd915f18d1)